extension/e2e-tests/loadAccount.test.ts

```diff
@@ -27,7 +27,10 @@ test("Load accounts on standalone network", async ({ page, extensionId }) => {
   });
   await expect(page.getByTestId("account-assets")).toContainText("XLM");
 });
-test("Switches account with password prompt", async ({ page, extensionId }) => {
+test("Switches account without password prompt", async ({
```

Comment:
fixing a typo in this test name while I'm here - we're making sure the switching account doesn't prompt you for a password (like it used to before)